### ISSUE TITLE
ScrollBar/Indicator not visible when used with Accessibility White

### ISSUE THREAD

**Issue by chigy** [microsoft/microsoft-ui-xaml#2727]
**Describe the bug**
When I turn on Accessibility White, ScrollBar is not visible for two reasons:

1. ScrollIndicator doesn't display until I hover over it (on regular version, scroll indicator is always visible).
2. The ScrollBar's handle is white.

**Steps to reproduce the bug**
I used Controls Gallery and turned Accessibility setting to White.

**Expected behavior**
I can see and use ScrollBar easily.

**Screenshots**
![image](https://user-images.githubusercontent.com/22958680/85315586-306ce080-b470-11ea-97cc-434dab26f72a.png)

**Version Info**
Build 20152.rs_prerelease.200617-1502

NuGet package version: 
Whatever the version latest XCG uses.

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (20152)              | Yes    |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Comment by StephenLPeters** [comment]
We should also look at HighContrast Black and make sure there aren't issues there as well.

**Comment by StephenLPeters** [comment]
@ranjeshj fixing this will require pulling the template out of the system into winui 2 but @chigy says that work has benefits for some other initiatives as well.

**Comment by ranjeshj** [comment]
That seems reasonable thing to do given we have several updated styles in WinUI2. @RBrid  as FYI in case he has any concerns doing this.

**Comment by chingucoding** [comment]
@StephenLPeters @ranjeshj What steps are needed to move the template inside WinUI 2? Given that this is an accessibility issue which we could solve easily just with the ScrollViewer template, this seems reasonable to think about now.

**Comment by StephenLPeters** [comment]
It would be the Scrollbar template that would need pulling out right?  @llongley I think you did a lot of that work right? would it be easy enough?

**Comment by chingucoding** [comment]
Yes, I think for this specific case, it would be the ScrollBar which turns out is already in WinUI. I'll take a look at this issue then, if that's fine.

**Comment by StephenLPeters** [comment]
Huh, for some reason I thought that hadn't been lifted out. Cool! yeah I think this is actionable in winui2
